tests/chaos/oadp/conftest.py

```diff
@@ -64,3 +74,14 @@ def rebooted_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress, wo
     LOGGER.info(f"Waiting for node {vm_node.name} to come back online")
     wait_for_node_status(node=vm_node, status=True, wait_timeout=TIMEOUT_10MIN)
     return
+
+
+@pytest.fixture()
+def drain_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
+    vm_node = rhel_vm_with_dv_running.vmi.node
+    # Ensure that backup has been in the InProgress status and then drain the node
+    _ = oadp_backup_in_progress
+    time.sleep(TIMEOUT_5SEC)
```

Comment:
you already wait for the INPROGRESS status in oadp_backup_in_progress fixture

Reply:
From the timestamps, these two fixtures run almost back-to-back. The node drain may start before the backup actually begins copying data, so a brief sleep was added to ensure proper sequencing.